trim.h

```diff
@@ -25,7 +25,14 @@
 #include "str.h"
 
 /* whitespace */
-#define is_ws(c) ((c) == ' ' || (c) == '\r' || (c) == '\n' || (c) == '\t')
+static inline int
+is_ws(unsigned char ch)
+{
+    const unsigned int mask = (1 << (' ' - 1)) | (1 << ('\r' - 1)) |
```

Comment:
Notice you have diverged from the overall coding style of the file on two levels:

* Windows-style function type/name alignment (2 lines), instead of Linux-style (1 line)
* 4 spaces instead of tab for alignment (98%+ of OpenSIPS code uses tabs)